spec/controllers/api/v1/accounts/relationships_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::Accounts::RelationshipsController do
```

Comment:
FYI - we are in the progress of (Slowly!) migrating much of the api request coverage from controller specs to request specs. I think this one (/api/v1/accounts/relationships) has already been moved and you could add this coverage to that spot, instead of re-adding this controller spec?